server/belga/io/feeding_services/twitter_belga.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+import re
+from datetime import datetime
+
+import requests
+
+import superdesk
+from superdesk.errors import IngestTwitterError, SuperdeskIngestError
+from superdesk.io.feeding_services import TwitterFeedingService
+from superdesk.io.registry import register_feeding_service
+from superdesk.metadata.item import GUID_FIELD
+
+
+class IngestTwitterBelgaError(SuperdeskIngestError):
+    _codes = {
+        6300: "Invalid iframely api key"
+    }
+
+    @classmethod
+    def TwitterInvalidIframelyKey(cls, exception=None, provider=None):
+        return IngestTwitterBelgaError(6300, exception, provider)
+
+
+class TwitterBelgaFeedingService(TwitterFeedingService):
+    NAME = 'twitter_belga'
+
+    label = 'Twitter Belga'
+
+    ERRORS = [
```

Comment:
We use list that match with format of class `TwitterFeedingService` in `core`. It will raise error in core if we change to tuple, to resolve this we have to fix many code in `core`. So we still keep list type for ERRORS, is it okay for this?